tests/unit/tests_mvd3.cpp

```diff
@@ -427,7 +425,7 @@ BOOST_AUTO_TEST_CASE( mvdTsvFiles )
     const TSV::MEComboEntry& info = TSVInfos[9];
 
     BOOST_CHECK_EQUAL(info.morphologyName, "87dd39e6b0255ec053001f16da85b0e0");
-    BOOST_CHECK_EQUAL(info.layer, 1);
+    BOOST_CHECK_EQUAL(info.layer, "1");
```

Comment:
Looks like the equivalent to this still should be fixed in the Python test?

Reply:
Oh right, good catch! I missed the python test. 
Hope it will pass this time.